resources/build/bootstrap-macos-11.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# This bootstrap script is used by both the Github CI action and the
+# Vagrant VM configuration.
+
+set -xeo pipefail
+
+# macOS 11 runners ship with clang 13. Conan Center only has pre-built
+# binaries for clang 12. This allows packages to be built locally.
```

Comment:
[question] Is this the case in general, or specifically the cpython package?

Reply:
cpython, and many of its dependencies.